packages/mui-material/src/TableCell/TableCell.js

```diff
@@ -140,7 +140,12 @@ const TableCell = React.forwardRef(function TableCell(inProps, ref) {
   }
 
   let scope = scopeProp;
-  if (!scope && isHeadCell) {
+  // https://github.com/mui/material-ui/issues/35031#issuecomment-1359346668
```

Comment:
Let's not link GitHub comment. The comment below to explain the reason is enough.